extension/js/common/core/pgp.ts

```diff
@@ -182,22 +182,10 @@ export class Pgp {
   }
 
   public static internal = {
-    msgBlockObj: (type: MsgBlockType, content: string | Buf, missingEnd = false): MsgBlock => ({ type, content, complete: !missingEnd }),
-    msgBlockDecryptErrObj: (encryptedContent: string | Buf, decryptErr: DecryptError): MsgBlock => ({ type: 'decryptErr', content: encryptedContent, decryptErr, complete: true }),
     msgBlockAttObj: (type: MsgBlockType, content: string, attMeta: AttMeta): MsgBlock => ({ type, content, complete: true, attMeta }),
```

Comment:
One day I may move these into a separate class, but I have to think about it first.

MsgBlockFactory or something like that. But if I did that, there would be a whole bunch of other things I'd have to refactor (move things from `PgpArmor` and `Mime`) and I'm not 100% sure it makes sense yet.

I'll make an issue.